### PR TITLE
UI for scheduled changes in Releases

### PR DESCRIPTION
**Pull request by NinadBhat**
https://bugzilla.mozilla.org/show_bug.cgi?id=1310214

### CODE REVIEW

**Comment by mozbhearsum** [comment]
Can you update this against the latest version of #196? It looks like it's a week or so behind.

File: ui/app/js/controllers/release_scheduled_change_edit_controller.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+  };
+
+  $scope.changeName = function () {
+    //wait for actual file to be loaded
+    setTimeout($scope.fillName, 0);
+  };
+
+  $scope.saveChanges = function () {
+    if (!$scope.sc.product.trim()) {
```
Comment (NinadBhat) [inline ui/app/js/controllers/release_scheduled_change_edit_controller.js]: I have created release_scheduled_change_delete_controller.js  for Scheduled Deletion

File: ui/app/js/controllers/release_scheduled_changes_controller.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+      },
+    ];
+  } else {
+  $scope.ordering_options = [
+    {
+      text: "When",
+      value: "when"
+    },
+    {
+      text: "Product, Name",
```
Comment (mozbhearsum) [inline ui/app/js/controllers/release_scheduled_changes_controller.js]: Close! These should be separate items though. There's really no reason to have a two-part sort, but sorting on either exclusively is useful.

File: auslib/admin/views/scheduled_changes.py [hunk 3]
```diff
@@ -82,7 +82,7 @@ def _post(self, sc_id, form, transaction, changed_by):
         for k, v in form._fields.iteritems():
             # sc_data_version is a "special" column, in that it's not part of the
             # primary data, and shouldn't be updatable by the user.
-            if k == "sc_data_version":
+            if k == "sc_data_version" or v.data is None:
```
Comment (mozbhearsum) [inline auslib/admin/views/scheduled_changes.py]: Unfortunately, this has made it impossible to update values to None in the Rules Scheduled Changes :(. Eg: if I scheduled a change, then try to update that scheduled change to change "product" to an empty value, it accepts the update but doesn't make the change. I _think_ the right thing to do here is to make sure the UI doesn't set "data" in the request if no new file is being uploaded - the if block below this should take care of handling that correctly.

Sorry for misleading you by suggesting modifying this yesterday =/.

**Pull request merged**